Header header

int8 red_light

int8 yellow_light

int8 green_light